cmd/zedmanager/updatestatus.go

```diff
@@ -1035,7 +1039,7 @@ func doInactivate(ctx *zedmanagerContext, uuidStr string,
 				changed = true
 			}
 		}
-		return changed
+		return changed, done
```

Comment:
done, is never turned true ???. Is that what you expected.

Reply:
It is on line 1089, but then it isn't used in the return statement on line 1093. I'll fix the return to be 
 return changed, done
so that the code makes more sense.